app/emails/consent/invite.njk

```diff
@@ -1,43 +1,52 @@
 {% else %}
-Your local health team are coming to {{ session.location.name }} on {{ session.summary.dates }} to give the following vaccines:
+We’re coming to {{ session.location.name }} on {{ session.summary.dates }} to offer the following vaccines to your child:
 
 {% for programme in session.programmes %}
 - {{ programme.information.title }}
 {%- endfor %}
 {% endif %}
 
-## What the {{vaccine__n}} {{is__n}} for
+We would like your consent to vaccinate {{ consent.child.fullName }}. You can give this by filling in our online form (the link is below).
 
 {% for programme in session.programmes %}
+## About the {{ programme.name | replace("Flu", "flu") }} vaccine
+
 {{ programme.information.description }}
+
+[Find out more about the {{ programme.name | replace("Flu", "flu") }} vaccine on NHS.UK]({{ programme.information.url }})
+
+{% if programme.information.leaflet %}
+[Read the patient information leaflet]({{ programme.information.leaflet }})
+{% endif %}
 {% endfor %}
 
-## Please give or refuse consent
+## How to respond
 
-If you want your child to get {% if session.programmes.length == 1 %}this vaccine{% else %}these vaccines{% endif %} at school, you need to give consent by {{ session.formatted.firstDate }}.
+> It’s important to let us know whether you do or do not want your child to have {% if session.programmes.length == 1 %}this vaccination{% else %}these vaccinations{% endif %}. It will take less than 5 minutes to respond using the link below.
 
-[Give or refuse consent for the {{ session.vaccination }}]({{ session.consentUrl }})
+[Respond to the consent request]({{ session.consentUrl }})
 
-Responding will take less than 5 minutes. If you do not give consent, it’s still important for you to let us know.
+You need to respond by {{ session.formatted.firstDate }}.
 
-## Your data
+{% if programme.type != ProgrammeType.Flu %}
+## Talk to your child about what they want
 
-By responding, you’re agreeing to your data being processed as set out in our [privacy notice]({{ data.organisation.privacyPolicyUrl }}).
+We suggest you talk to your child about the {{ vaccine__n }} before you respond to us.
+{% endif %}
 
-{% for programme in session.programmes %}
-## About the {{ programme.name | replace("Flu", "flu") }} vaccine
+## If you cannot use the online form
 
-{{ programme.information.audience }}
+If you cannot use the online form, you can respond over the phone using the contact details below.
 
-[Find out more about the {{ programme.name | replace("Flu", "flu") }} vaccine on NHS.UK]({{ programme.information.url }})
+## Your data
 
-{% if programme.information.leaflet %}
-[Read the patient information leaflet]({{ programme.information.leaflet }})
-{% endif %}
+By responding, you’re agreeing to your data being processed as set out in our [privacy notice]({{ data.organisation.privacyPolicyUrl }}).
 
 {% if programme.type == ProgrammeType.Flu %}
+## Something here
```

Comment:
Flu emails contain additional information about Gelatine content, and the option to have an injection instead. I’ve left this after the link to give or refuse consent, but it needs a title, not that we’ve moved the rest of the content above the link.

Reply:
Have updated this to say ‘About the nasty spray’ for now.